test/gurka/test_search_filter.cc

```diff
@@ -261,7 +261,7 @@ class ExcludeClosuresOnWaypoints : public ::testing::TestWithParam<std::string>
             |
             H
 
-   L-4---5-M-6-N
+   L4----5-M-6-N
```

Comment:
this was evil. we have a test that checks if trivial routes fail when the edge is closed. the trivial route is from `4` to `5`. we close the edge `LM`. on amd64 `4` snaps to `A` above and `5` snaps to `M`. since they are in disconnected components the route fails as you expect. on arm64 `4` snaps to `M` and since `5` also snaps to `M` the route passes and the test fails. this is because, on both platforms, `4` is equidistant to `M` and to `A` above. so it was just some floating point noise that made it pick one over the other. by moving `4` to the left one char, it becomes closer to `A` regardless of platform and floating point noise and so we get reliable results.